### PR TITLE
Home page feature

### PR DESCRIPTION
**Pull request by rnagulapalle**
this is opened for implementing set as home page feature

### CODE REVIEW

File: controllers/selectMainRoute.js [hunk 1]
```diff
@@ -23,26 +24,45 @@
...
+        $scope.setAsHomePage = function (model,$event) {
+            var confirm = $mdDialog.confirm()
+                .title('Set As Home Page')
+                .textContent('Are you sure you want to set this current page as your main home page that people land on when they visit your domain ?')
+                .targetEvent($event)
+                .ok('Confirm')
+                .cancel('Cancel');
+            $mdDialog.show(confirm).then(function() {
+                model.data.main = true;
+                console.log(model);
```
Comment (alexgurrola) [inline controllers/selectMainRoute.js]: If you're going to use `console.log`, you should wrap it in a `if (!Stratus.Environment.get('production'))` so it only shows in `dev` or `prev` environments.

File: controllers/selectMainRoute.js [hunk 2]
```diff
@@ -23,26 +24,45 @@
...
           $scope.routes = routing
           angular.forEach($scope.routes, function (route) {
             if (route.main) {
               $scope.mainRoute = route.id
+
             }
+              route.homepage = $scope.homepage;
           })
         }
+        console.log($scope.routes);
```
Comment (alexgurrola) [inline controllers/selectMainRoute.js]: Same console issue as above.

File: controllers/selectMainRoute.js [hunk 3]
```diff
@@ -23,26 +24,45 @@
...
       })
 
       $scope.update = function () {
         angular.forEach($scope.routes, function (route) {
+            route.homepage = $scope.homepage;
+            console.log(route);
```
Comment (alexgurrola) [inline controllers/selectMainRoute.js]: Same console issue as above.

File: controllers/selectMainRoute.js [hunk 4]
```diff
@@ -23,26 +24,45 @@
...
           route.main = (route.id === $scope.mainRoute)
         })
+
+          console.log($scope.routes);
```
Comment (alexgurrola) [inline controllers/selectMainRoute.js]: Same console issue as above.

**Comment by rnagulapalle** [comment]
Once you approve I will rebase my changes..

**Comment by alexgurrola** [comment]
This doesn't pass Standard JS requirements, so Travis is Failing.  As soon as Travis is successful, then you can be approved.  A portion of the issue is your extra spaces for indentation.

**Pull request merged**